dbt_subprojects/hourly_spellbook/models/_metrics/dune_index/metrics_dune_index_stats.sql

```diff
@@ -0,0 +1,163 @@
+{{ config(
+        schema = 'metrics'
+        , alias = 'dune_index_stats'
+        )
+}}
+
+with source as (
+    -- daily dune index per blockchain
+    select
+        blockchain
+        , block_date
+        , fees_index
+        , transfers_index
+        , tx_index
+        , dune_index
+    from
+        {{ ref('metrics_dune_index_daily') }}
+), current_day as (
+    select
+        blockchain
+        , ((fees_index + transfers_index + tx_index) / 3) as last_1_days_dune_index_contribution
+        , ((fees_index + transfers_index + tx_index) / 3) / dune_index as last_1_days_dune_index_contribution_percent
+        , dune_index as last_1_days_dune_index
+    from
+        source
+    where
+        block_date >= date_trunc('day', now()) - interval '1' day
+        and block_date < date_trunc('day', now())
+), previous_day as (
+    select
+        blockchain
+        , ((fees_index + transfers_index + tx_index) / 3) as previous_1_days_dune_index_contribution
+        , ((fees_index + transfers_index + tx_index) / 3) / dune_index as previous_1_days_dune_index_contribution_percent
+        , dune_index as previous_1_days_dune_index
+    from
+        source
+    where
+        block_date >= date_trunc('day', now()) - interval '2' day
+        and block_date < date_trunc('day', now()) - interval '1' day
+), daily_stats as (
+    select
+        c.blockchain
+        , c.last_1_days_dune_index_contribution
+        , c.last_1_days_dune_index_contribution_percent
+        , c.last_1_days_dune_index
+        , p.previous_1_days_dune_index_contribution
+        , p.previous_1_days_dune_index_contribution_percent
+        , p.previous_1_days_dune_index
+        , ((c.last_1_days_dune_index - coalesce(p.previous_1_days_dune_index, 0)) / coalesce(p.previous_1_days_dune_index, 1)) * 100 AS daily_percent_change
```

Comment:
should this percent change calculation be on the `dune_index` value itself, or the `dune_index_contribution`?